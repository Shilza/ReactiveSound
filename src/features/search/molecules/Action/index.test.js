import React from "react";
import {render} from 'react-testing-library';
import {Action} from "./index";

describe('<Action/> search feature molecule', () => {
    afterEach(() => {});

    const labelText = 'label';
    const {container, getByTestId} = render(
        <Action label={labelText}>
            <div>Children</div>
        </Action>
    );

    it('should have two child', () => {
        expect(container.firstChild.childNodes.length).toBe(2);
    });

    it('should have correct label', () => {
        expect(getByTestId('actionLabel')).toHaveTextContent(labelText);
    });
});