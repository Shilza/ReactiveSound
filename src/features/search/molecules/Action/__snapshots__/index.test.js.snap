// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test search Action should renders correctly 1`] = `
<div
  class="container"
>
  <div>
    Children
  </div>
  <span
    class="label"
    data-testid="actionLabel"
  >
    label
  </span>
</div>
`;
